README.md

```diff
@@ -349,6 +365,34 @@ access token only once, after which it becomes invalid.
   - [Get Current User's Profile](examples/data/users_profile/GetCurrentUsersProfileExample.java)
   - [Get a User's Profile](examples/data/users_profile/GetUsersProfileExample.java)
 
+## Spring Boot Example
+Spring Boot is an open-source Java framework that can be used to create production-grade web-based applications with very minimal configuration. It is built on top of the Spring framework which offers comprehensive infrastructure support for the development of Java web applications. This specific example provides users with a potential approach to configuring and instantiating the Spotify API Wrapper.
+
+1. Navigate to the **[Spring Initialzr](https://start.spring.io/)**
+2. Next, set **Project** to Maven, and language to Java. The remaining fields can be filled according to your preference.
+3. Add the **Spring Web** dependency (this will allow for API development using Spring MVC).
+4. Click **GENERATE** to create a zip file of the project and open it in your IDE of choice.
+5. Navigate to `src/main/java/com.package` and create a new folder called `Config`.
+6. Within this folder, a singleton instance (bean) of the SpotifyApi class is instantiated. 
+```
+@Configuration
+public class SpotifyConfig {
+    private static final String clientId = "clientId";
+    private static final String clientSecret = "clientSecret";
+    private static final URI redirectUri = SpotifyHttpManager.makeUri("path/to/redirect/URI");
+
+    @Bean
+    public SpotifyApi spotifyApi(){
+        return new SpotifyApi.Builder()
+                .setClientId(clientId)
+                .setClientSecret(clientSecret)
+                .setRedirectUri(redirectUri)
+                .build();
+    }
+}
+```
+7. This instantiated object can then be used throughout the application.
+
```

Comment:
I'm not a fan of adding length to the already large README by adding instructions for some specific Java framework. Also, isn't the only addition in that example `@Configuration`? I think the above general usage instructions are enough. This project's README should not teach how to setup a project in a Java framework.